modules/javafx.base/src/test/java/test/javafx/util/converter/LocalDateTimeStringConverterTest.java

```diff
@@ -56,11 +57,9 @@ public class LocalDateTimeStringConverterTest {
 
     private static final DateTimeFormatter aFormatter = DateTimeFormatter.ofPattern("dd MM yyyy HH mm ss");
     private static final DateTimeFormatter aParser = DateTimeFormatter.ofPattern("yyyy MM dd hh mm ss a");
+    private static Locale oldLocale;
```

Comment:
Isn't the creation of the DateTimeFormatter using the default locale? If so, this should probably be done after thr locale is set.

Reply:
Maybe JUnit5 can help you here. 
You can check out my recently merged PR for an example for the usage of the parameterized api introduced in JUnit5.
https://github.com/openjdk/jfx/pull/936

Reply:
I guess they fixed this rather irritating "feature" in JUnit5. Another, possibly less-intrusive, approach would be to initialize all of the static fields in a `static { ... }` block (with a comment as to why you can't use an `@BeforeClass` method.